bliss/encoder/plotting.py

```diff
@@ -112,3 +112,111 @@ def plot_detections(images, true_tile_cat, est_tile_cat, margin_px, ticks=None,
 
     fig.tight_layout()
     return fig
+
+
+def plot_maps(images, true_tile_cat, est_tile_cat, figsize=None):
```

Comment:
Ideally case studies would be able to define new plotting functions like this one within a case study subdirectory, rather than in the bliss package. These plotting routines would be passed to the encoder when instantiating it, like how metrics are handled already.

Reply:
This will be handled in a separate PR.